repos/spack_repo/builtin/packages/papi/package.py

```diff
@@ -70,20 +72,22 @@ class Papi(AutotoolsPackage, ROCmPackage):
     # The PAPI configure option "--with-shlib-tools" is deprecated
     # and therefore not implemented here
 
-    depends_on("c", type="build")  # generated
-    depends_on("cxx", type="build")  # generated
-    depends_on("fortran", type="build")  # generated
+    depends_on("c", type="build")
+    depends_on("cxx", type="build")
+    depends_on("fortran", type="build")
 
     depends_on("lm-sensors", when="+lmsensors")
     depends_on("cuda", when="+cuda")
     depends_on("cuda", when="+nvml")
     depends_on("bc", when="+cuda", type="build")
     depends_on("hsa-rocr-dev", when="+rocm")
     depends_on("rocprofiler-dev", when="+rocm")
+    depends_on("rocprofiler-sdk", when="+rocp_sdk")
     depends_on("llvm-amdgpu", when="+rocm")
     depends_on("rocm-openmp-extras", when="+rocm")
     depends_on("rocm-smi-lib", when="+rocm_smi")
 
+    conflicts("+rocp_sdk", when="@:7.1.0")
```

Comment:
Does this need a conflict between `+rocm` and `+rocp_sdk`? https://github.com/spack/spack-packages/pull/535/files